_shared/project/.github/workflows/deploy.yml

```
@@ -0,0 +1,38 @@
+name: Deploy
+concurrency:
+  group: deploy
+  cancel-in-progress: true
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+jobs:
+  CI:
+    uses: ./.github/workflows/ci.yml
+  Docker_Hub:
+    name: Docker Hub
+    needs: CI
```

```suggestion
  docker_hub:
    name: Docker Hub
    needs: [ci]
```

Using the brackets here isn't essential but would make this section look more like those generated from the JSON stuff below.

---

:+1: https://github.com/hypothesis/cookiecutters/pull/95/commits/a0e1ea391ab6524864dba1e0d0404b5984cb46f0